server/models/gateway.js

```diff
@@ -1,13 +1,20 @@
 var async = require('async');
+var crypto = require('crypto');
+var observerHelper = require('./observerHelper');
 
 module.exports = function extendGateway(Gateway) {
-  // Refer to server-service.js for description of instance vs currentInstance
-  // vs data.
+  Gateway.definition.properties.token.default = function() {
+    return crypto.randomBytes(24).toString('hex');
```

Comment:
maybe worth putting in a util file, so at least the 24 is shared among the various default functions.